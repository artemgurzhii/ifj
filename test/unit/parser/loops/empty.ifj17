do while foo > 0
loop
