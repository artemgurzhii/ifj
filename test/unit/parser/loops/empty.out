(while (> (id foo) (int 0))
)
